build/filter_test.go

```diff
@@ -230,20 +230,38 @@ var _ = Describe("Test filters utilities:", func() {
 	Describe("Test platform network filters", func() {
 		deployment := Deployment{}
 		deployment.PlatformNetworks = make([]*v1.PlatformNetwork, 0)
+		deployment.AddressPools = make([]*v1.AddressPool, 0)
 		coreNetworkFilter := NewCoreNetworkFilter()
-		var get_platform_network = func(nwk_type string) *v1.PlatformNetwork {
+		var get_platform_network = func(nwk_type string, associatedAddressPool []string) *v1.PlatformNetwork {
 			spec := v1.PlatformNetworkSpec{
-				Type: nwk_type,
+				Type:                   nwk_type,
+				AssociatedAddressPools: associatedAddressPool,
 			}
 			new_plat_nwk := v1.PlatformNetwork{}
+			new_plat_nwk.Name = "oam"
 			new_plat_nwk.Spec = spec
 			return &new_plat_nwk
 		}
+		var get_associated_address_pool = func(subnet string) *v1.AddressPool {
+			spec := v1.AddressPoolSpec{
+				Subnet: subnet,
+			}
+			new_address_pool := v1.AddressPool{}
+			new_address_pool.Name = "oam"
```

Comment:
Can you drive `Name` through variable instead of using hardcoded `oam`?

Reply:
Done